### ISSUE TITLE
Move webserver code from __init__ to cli.py

### ISSUE THREAD

**Issue by Uzay-G** [archivy/archivy#184]
Some of the code in `__init__.py` is only useful to prepare the webserver, and should be moved to the `run` command in `cli.py`. I'm not sure what would be the best way to do this, but it should be quite simple and might increase latency for non-web related commands.

This is the code that could be removed:
https://github.com/archivy/archivy/blob/b6fc3b25c4e66a0845219a537cec3d4adb31e5fe/archivy/__init__.py#L39-L55

**Comment by Uzay-G** [comment]
I'd like to hear your opinon on this @srevinsaju

**Comment by srevinsaju** [comment]
I agree. This would be a great improvement. I actually, would say that the `__init__.py` should actually not contain any specific code. But, we can start little, and move everything to their respective places, and not keep a lot of stuff on `__init__.py`. This would be a great improvement overall. Thanks

**Comment by srevinsaju** [comment]
I have a few commits on my local branch, it tries to make archivy more OOP, but I was not able to complete it. Would you like to take a look?

**Comment by Uzay-G** [comment]
Ok

**Comment by srevinsaju** [comment]
I commit everything which was not staged on https://github.com/srevinsaju/archivy/tree/develop as a last commit, which is a bit huge. I dunno if it could help, but you know better, if its a small thing which needs a fix. Let me know too. (PS: it doesnt work out of the box yet)

**Comment by Uzay-G** [comment]
Whoa that is a large diff :sweat_smile:

**Comment by srevinsaju** [comment]
yea, :sweat:

Some notable commits, which could improve archivy would be 
* Move archivy base to flask's blueprint object: https://github.com/archivy/archivy/commit/e023886135572c269b2b336648671817bf559243

* Cleanup `archivy/__init__.py`:
https://github.com/archivy/archivy/commit/7cdc1405e0a9dabd0bca28ec6dcc397484f4fb5b

* Update requirements 
https://github.com/archivy/archivy/commit/ffb820bcd44ba26a647362734c425eb5c2685bee

* Add click and jinja:  https://github.com/archivy/archivy/commit/12175b64c3d89ed075f90d6d6d33089d90322af7

* Use black to format archivy. https://github.com/archivy/archivy/commit/e6ee1a7317c5eb36519eb7b0cc761e9338235f30

* Add a production `gunicorn` wrapper: https://github.com/archivy/archivy/commit/90e393bf05618d38becbae5127f557990415a250

**Comment by clemux** [comment]
What's the status of this issue? Do you know how to remove the `__init__.py` code and have not found the time for that yet? Or do you need help to figure it out?

**Comment by Uzay-G** [comment]
I haven't had time to do this yet, help is welcome.
